src/lib/core.js

```diff
@@ -7640,8 +7637,13 @@ export default function (context, pluginCallButtons, plugins, lang, options, _re
                 }
             }
 
+            core._cachingButtons();
+            core.history._resetCachingButton();
+
             if (core.hasFocus) event._applyTagEffects();
 
+            if (core.isReadOnly) util.setDisabledButtons(true, core.resizingDisabledButtons);

```

Comment:
There is no need in this line if you will apply the changes from https://github.com/JiHong88/SunEditor/pull/839 pull-request